### ISSUE TITLE
Supernet accuracy

### ISSUE THREAD

**Issue by xiaoshingshing** [megvii-model/SinglePathOneShot#8]
I train the supernet with the default setting, and got an accuracy of 

`TRAIN Iter 150000: lr = 0.000003,	loss = 2.805815,	Top-1 err = 0.417285,	Top-5 err = 0.202197,	data_time = 0.050527,	train_time = 0.565107`

Is this accuracy right?

**Comment by ZichaoGuo** [comment]
Our Top-1 error is about 0.4.
